202306-NEA-DZ-FEW/team-a
# 28 authentication setup

## authentication setup

###  Description

fixes #28

1. **Authentication Setup**: I have added the necessary files for the team to work on authentication. This includes the required configurations, components, and any relevant code changes.

2. **Bug Fixes and Enhancements**:
   - Resolved issues related to the language select menu.
   - Addressed console warnings and other issues related to i18next.
   - Improved the snapshot testing setup for i18next, ensuring it aligns with the project's requirements.

These changes will enhance the overall stability and functionality of the project, providing a better user experience and streamlining development for the team.

## Checklist

- [x] Appropriate documentation has been updated or added.
- [x] New tests have been added to cover these changes.
- [x] All issues mentioned in this PR description have been addressed.

## How to test this

To review these changes, please follow these steps:

1. Run `git fetch origin` to ensure you have the latest changes from the remote repository.
2. Check out the new branch for this PR by running `git checkout 28-authentication-setup`.
3. Run the command `npm run dev`.
4. Once you have the changes locally, experiment with the app

_keep in mind that the changes are affecting only these routes:  `/` , `/auth/sign-in` , `/auth/sign-up`, `/dashboard`_

Please review and provide your feedback on these changes. Your input is greatly appreciated!

# Related Issue

- Resolve #28
